terralab/commands/submit_commands.py

```diff
@@ -0,0 +1,34 @@
+# commands/submit_commands.py
+
+import click
+import logging
+
+from terralab.logic import submit_logic
+from terralab.utils import handle_api_exceptions, process_json_to_dict
+from terralab.logic import pipelines_logic
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument("pipeline_name")
+@click.option(
+    "--version", type=int, default=0, help="pipeline version; default: 0"
+)  # once TSPS-370 is done, remove default
+@click.option("--inputs", type=str, required=True, help="JSON string input")
+@click.option(
+    "--description", type=str, default="", help="optional description for the job"
+)
+@handle_api_exceptions
```

Comment:
see if we can instead subclass the commands that call APIs to have this decorator automatically (don't block on this)